clickqt/core/control.py

```diff
@@ -143,7 +149,8 @@ def parse(
             if len(cmd.params) > 0:
                 child_tabs = QWidget()
                 child_tabs.setLayout(QVBoxLayout())
-                group_params = self.parse_cmd(cmd, concat_group_names)
+                option_group = determine_option_group_presence(cmd)
```

Comment:
should be is_option_group, otherwise i would not expect a boolean (holds for the rest of the file as well)